snooty/types.py

```diff
@@ -64,8 +66,11 @@ class StaticAsset:
     fileid: FileId
     path: Path
     upload: bool
+    diagnostics: List[Diagnostic]
     _checksum: Optional[str]
     _data: Optional[bytes]
+    _width: Optional[float]
```

Comment:
I think it would be better for these to be a single field, e.g. `_dimensions: Optional[Tuple[float, float]]`, since they're assigned together and we can't have a width without a height or vice versa

Reply:
i actually had it as a single field previously (single float field for [aspect_ratio](https://github.com/mongodb/snooty-parser/pull/547/commits/ea329a7fdad2ed2dcc945570e3d7efa0b5613a11) ), but converted to width and height. good point on a tuple 👍 will change

Reply:
updated! but kept as two fields on the output directive to preserve output AST as width and height fields